### PR TITLE
Small, minor improvements to structure, efficiency, adherence to Java…

### PR DESCRIPTION
**Pull request by yakovliam**
… principles, etc

### CODE REVIEW

**Comment by lucian929** [comment]
Hey there, looking through this at the moment and I'm holding off on merging this because a lot of it is quite advanced for what this plugin is at the moment

File: pom.xml [hunk 1]
```diff
@@ -5,7 +5,7 @@
     <modelVersion>4.0.0</modelVersion>
 
     <groupId>me.raindropz.mc</groupId>
-    <artifactId>npcommand</artifactId>
+    <artifactId>NPCommand</artifactId>
```
Comment (mbax) [inline pom.xml:8]: Changing artifactId after its already existed for a while is an odd choice. Capitalizing it is an even odder choice.

File: src/main/java/me/raindropz/mc/NPCommand.java [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+    }
+
+    private void logPluginStateChange(boolean isStartup) {
+        PluginDescriptionFile descriptionFile = this.getDescription();
+        String version = descriptionFile.getVersion();
+        String author = descriptionFile.getAuthors().stream()
+                .findFirst()
+                .orElse("Raindropz");
+
+        this.getLogger().info("NPCommand v" + version + "by " + author + " is now " + (isStartup ? "enabled" : "disabled") + ".");
```
Comment (Sancires) [inline src/main/java/me/raindropz/mc/NPCommand.java:59]: Rather unnecessary addition for a plugin of this size

Comment (yakovliam) [reply src/main/java/me/raindropz/mc/NPCommand.java:59]: This is a code-saving method which prevents duplication of code throughout the project

File: pom.xml (hunk 1 above)
Comment (yakovliam) [reply pom.xml:8]: The capitalization is to match the project name, which is a standard convention when building a project into an artifact

**Comment by lucian929** [comment]
Hey there, please create a new PR to the "testing" branch I created.

File: pom.xml (hunk 1 above)
Comment (LichtHund) [reply pom.xml:8]: https://maven.apache.org/guides/mini/guide-naming-conventions.html

> artifactId is the name of the jar without version. If you created it, then you can choose whatever name you **want with lowercase letters** and no strange symbols. If it's a third party jar, you have to take the name of the jar as it's distributed.
> eg. maven, commons-math

**Pull request closed without merging**